.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
-  "plugins": ["prettier-doc", "@typescript-eslint", "prettier"],
```

Comment:
I'm not sure about this change. I think eslint is failing because it's trying to run Prettier synchronously. It throws this error: `TypeError: prettier.resolveConfig.sync is not a function`

Reply:
right, maybe that's something to bring up to prettier? since I's expect that the authoring lint rules should continue to work in v3?

Reply:
The ESLint plugin probably doesn't support Prettier 3, that's fine. We should probably run Prettier in CI anyway

Reply:
FYI: `eslint-plugin-prettier` already supported Prettier v3 https://github.com/prettier/eslint-plugin-prettier/releases/tag/v5.0.0-alpha.0